ql/src/test/org/apache/hadoop/hive/ql/lockmgr/TestDbTxnManager.java

```diff
@@ -486,6 +505,8 @@ public void setUp() throws Exception {
     conf.setTimeVar(HiveConf.ConfVars.HIVE_TIMEDOUT_TXN_REAPER_START, 0, TimeUnit.SECONDS);
     conf.setTimeVar(HiveConf.ConfVars.HIVE_TXN_TIMEOUT, 10, TimeUnit.SECONDS);
     houseKeeperService = new AcidHouseKeeperService();
+    MetastoreConf.setTimeVar(conf, MetastoreConf.ConfVars.REPL_TXN_TIMEOUT, 1, TimeUnit.MINUTES);
```

Comment:
Do we need the test time interval to be a min? Or it can be lesser.

Reply:
It can be less